.github/workflows/scala-steward.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
could you add `workflow_dispatch` ? useful for ad hoc runs

Reply:
Done